models/user.js

```diff
@@ -353,7 +367,42 @@ exports.getUserInfo = function(req, callback = (user, isAdmin) => {})
 	})
 }
 
-exports.getUserProfileInfo = function (userid, callback = { found: () => { }, notFound: () => {} })
+exports.getUserProfileInfo = function(userid, callback = { found: () => {}, notFound: () => {} })
```

Comment:
can you fix this up in your other branch? 
FROM:
found: () => {} 
TO:
found: (profile, address, payment) => {}